readme.txt

```diff
@@ -2,7 +2,7 @@
 Contributors:      10up, jeffpaul, dkotter
 Tags:              AI, Artifical Intelligence, ML, Machine Learning, Microsoft Azure, IBM Watson, OpenAI, ChatGPT, Content Tagging, Classification, Smart Cropping, Alt Text
 Requires at least: 5.7
-Tested up to:      6.1
+Tested up to:      6.2

```

Comment:
@ggutenberg Can we check other places like `readme.txt`, conditional changes in code, etc, and upgrade them too?

Reply:
@ggutenberg I overlooked `readme.txt`, so we are good on this.

-----------

I did not see this information in the plugin header (main plugin file `classifai.php`). Can you add this?

You can read more about plugin header requirements here: https://developer.wordpress.org/plugins/plugin-basics/header-requirements/